### DISCUSSION TITLE
被取代掉的工作:(

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#89] [General]
從之前就一直聽過以後偏"人力"的工作會被機器取代掉，但究竟是那些，以及何時呢?
文章中提及約於十年後，因為AI及機器的發展，約有四分之一的人力會被取代。
而常見的產線作業員,售票員,清潔工,產線工程師,等等都為高機率被取代掉的工作。
雖然科技進步，但必然會做出犧牲-工作機會銳減。培養專業技能，不要輕易被機器取代!

文章來源:
https://udn.com/news/story/7241/5852191

**Comment by ccathyy** [comment]
我記得有一位教授在他的演講中提到
「培養跨領域」的才能
可以逃過「AI將取代許多工作」這個危難
持續精進自己
才不會被這個世界所淘汰!

**Comment by wh20502** [comment]
AI取代勞力型工作是已經有所耳聞，不過取代勞心型我覺得還久，因為勞心型的專業性不是那好模仿，因為實際狀況可能有很多變化，我的想法。

**Comment by ghost** [comment]
為了讓自己不被取代掉 多培養一些機器人比較辦不到的技能 對我來說才是防止被時代淘汰的對策

**Comment by Shiba010** [comment]
會不會在未來的世界，連醫生也會被ai取代了....

**Comment by jack90325** [comment]
雖然說要培養專業技能，但是有些制式化的工作真的很容易被取代，一定會大量失業
